src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
After thinking about it, we don't want to use _.throttle for notifyDirectOffersChanged, we need to use _.debounce (with a max wait time) - because throttle will fire on the *first* transaction per ledger that updates the orderbook, then not fire again until after the throttle time - we want to wait until the *last* transaction from an incoming ledger has been processed before further calculation.

The same might be true for computing autobridged offers - what happens when compute is triggered by leg book 1, then in the same ledger, leg book 2 is modified? In this implementation, wouldn't leg book 2's  modifications not be reflected in the orderbook until the next time computing is triggered, on a later ledger?